sqlglot/transforms.py

```diff
@@ -368,6 +368,25 @@ def _udtf_type(u: exp.Unnest, has_multi_expr: bool) -> t.Type[exp.Func]:
                 expression.args["joins"].remove(join)
 
                 alias_cols = alias.columns if alias else []
+
+                """
+                Handle Presto CROSS JOIN UNNEST to LATERAL VIEW EXPLODE for Multiple or No Exploded table column alias.
+                
+                Spark/Hive LATERAL VIEW EXPLODE requires only single alias for respective exploded array/struct to be given for unnest ulike trino/presto which can take multiple.
+                https://cwiki.apache.org/confluence/display/Hive/LanguageManual+LateralView
+                https://spark.apache.org/docs/latest/sql-ref-syntax-qry-select-lateral-view.html
+                
+                """
```

Comment:
[Nit] let's not use a multi-line string here, it's inconsistent with the existing style. Instead, use normal `#` comments.

Reply:
this is doable, i'll update.

Reply:
Updated with [b789e00](https://github.com/tobymao/sqlglot/pull/4339/commits/b789e00803367cf347246b71737104fd0ac56767)